basic/exercise4/script4.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
-function leapYear(year) {
-    return ((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0);
+function isLeapYear(year) {
+    return ((year % 4 === 0) && (year % 100 != 0)) || (year % 400 === 0);
```

Comment:
Please also use !== instead of !=
Please read what is the difference - similar to == vs ===